### ISSUE TITLE
Wallet - Asset list - add test ERC20 token

### ISSUE THREAD

**Issue by sla-shi** [status-im/status-react#1951]
### User Story

As a developer, I want to have the test ERC20 token listed in new wallet to breakdown the development of the asset management story #1722. 

Extends the Wallet Main screen story #1523.
See also #1654

#### Acceptance
- [ ] I can see the test token balance in the asset list

#### Removed from scope
- I can refresh the test token value by pull-to-refresh gesture
- I can see the current recent total market value of my assets (ETH and the test ERC20 token) 

![wallet-assets.png](https://images.zenhubusercontent.com/5975f08f61b4ca3b0a757622/668dd124-b931-46eb-a6df-22a1cdad0f92)

**Comment by kylerchin** [comment]
Awesome! Well done! @sla-shi

**Comment by goranjovic** [comment]
I only need the first point to unblock me from further work - Just need something tappable to be able to reach the token details screen

**Comment by kylerchin** [comment]
@goranjovic yes, ERC20 is important. Would you like help on coding the pull to refresh feature?

**Comment by goranjovic** [comment]
Hey @kylerchin thanks for reaching out! I don't in this specific moment (still mocking up stuff), but definitely could use any help at some point going forward and definitely thanks for that! :)

Also, if you'd like to push your own contributions to the project, definitely let us know!

**Comment by andytudhope** [comment]
@kylerchin you can always check which issues currently have bounties attached to them here: https://openbounty.status.im/ - we'd love you to push your own contributions even if Goran is currently on top of this particular issue. Also, feel free to join the community in chat.status.im for more info and friends, yay! 🎉

**Comment by oskarth** [comment]
@goranjovic @sla-shi Please advice if this issue can be closed or not. If not, please move it to the appropriate column.

**Comment by goranjovic** [comment]
The main point is done.
The two other points (removed from scope) are superseded by #2466 and #2468 
We can definitely close this one since the merged PR covers the main point completely
